libmscore/textedit.cpp

```diff
@@ -245,7 +245,7 @@ bool TextBase::edit(EditData& ed)
             }
 
       if (!wasHex) {
-//printf("======%x\n", s.isEmpty() ? -1 : s[0].unicode());
+            //printf("======%x\n", s.isEmpty() ? -1 : s[0].unicode());
```

Comment:
I'd rather don't indent here. That way this debugging artifact (literally) sticks out better

Reply:
@Jojo-Schmitz as you wish :-)